docs/UserGuide.adoc

```diff
@@ -328,6 +328,44 @@ There is no need to synchronize manually.
 Stops synchronizing of contact information with any previously configured cloud accounts. +
 Format: `unsync`
 
+== Facebook Integration [Coming in v2.0]:
+
+=== Connecting to Facebook: `connectfacebook`
+
+Synchronizes your AddressBook with your personal Facebook account. +
+Format: `connectfacebook`
+
+****
+* Launches a Facebook authorization page in the browser
```

Comment:
I think for user guide would be great if we can tell the user what it means to them. Perhaps something like:

You will see a facebook authorization page displayed on the browser. Enter your credentials and confirm blah blah blah to allow KayPoh to access your facebook contact information.